x-pack/plugins/security_solution/scripts/endpoint/README.md

```diff
@@ -13,11 +13,11 @@ Example command sequence to get ES and kibana running with sample data after ins
 
 `yarn es snapshot` -> starts ES
 
-`npx yarn start --xpack.securitySolution.endpoint.enabled=true --no-base-path` -> starts kibana
+`npx yarn start --xpack.ingestManager.enabled=true --xpack.ingestManager.epm.enabled=true --xpack.ingestManager.fleet.enabled=true --no-base-path --verbose` -> starts kibana with Ingest and Fleet enabled
```

Comment:
It is my understanding that only one of the fleet properties is needed as the others have been combined into one. They all still work, but using one was done for convenience.

Reply:
thats the right idea @nnamdifrankie - we can leave the last 2 options out.  For some detail they are still available if one wished to 'disable' them in some combination (tho I dont know what customer relating scenario that would mimic if any) - but they default to 'enabled' and are implied if the ingestManager.enabled is set.